installers/linux/shared/partials/_go-server-config-migration.sh.erb

```diff
@@ -22,6 +22,8 @@
 
             . "/etc/default/<%= name %>"
 
+            mv "/etc/default/<%= name %>" "/etc/default/<%= name %>.pre-migration"
```

Comment:
For anyone wondering, we had a similar line in agent migration, but we missed it in the server migration.

This introduces another bug where a server upgrade from 19.6->19.7->19.8 might keep doing the same migration again and again.